pkg/apis/operator/v1alpha1/operandregistry_types.go

```diff
@@ -80,13 +80,17 @@ type OperandRegistrySpec struct {
 // OperandRegistryStatus defines the observed state of OperandRegistry
 // +k8s:openapi-gen=true
 type OperandRegistryStatus struct {
+	// Phase is the operator running phase
```

Comment:
This is an overall operator operating status, we should have  a clearer description for this.